core/templates/dev/head/player/flag_exploration_popup.html

```diff
@@ -0,0 +1,42 @@
+<script type="text/ng-template" id="modals/flagExploration">
+  <div class="modal-header">
+    <h3>
+      <span>Report Exploration</span>
```

Comment:
Flag Exploration for Moderator Review

We also need to internationalize this, I think.